README.md

```diff
@@ -188,3 +188,6 @@ Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Con
 
 ## Contact
 You can reach the admins, maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the OSP project, you may join the stream [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) and ask questions or intereact with the community. Join Us!
+
+## LICENSE
+Open Source Programs Backend is licensed under the GNU General Public License v3.0. To know more about it you can read   [GPL v3](https://github.com/anitab-org/open-source-programs-backend/blob/develop/LICENSE) guidelines.
```

Comment:
```suggestion
Open Source Programs Backend is licensed under the GNU General Public License v3.0. To know more about it you can read   [GPL v3](LICENSE) guidelines.
```

relative paths are better, so it will work even if repo name changes or you go to your fork ;)